clusterloader2/go.mod

```diff
@@ -0,0 +1,72 @@
+module k8s.io/perf-tests/clusterloader2
+
+go 1.12
+
+replace k8s.io/api => k8s.io/api v0.16.4
```

Comment:
Forgive me my godeps ignorance, but why those are v0...?
Shouldn't those directly reflect the k8s releases, and thus be 1.16.4?

Reply:
k8s.io/api follow the same tagging scheme as client-go: https://github.com/kubernetes/client-go#compatibility-your-code---client-go